.tests/compatibility/Snakefile

```diff
@@ -0,0 +1,45 @@
+from hydra_genetics.utils.resources import load_resources
+
+
+rule all:
+    input:
+        "qc/multiqc/multiqc.html",
+
+
+configfile: "config.yaml"
+
+
+config = load_resources(config, config["resources"])
+
+
+def extract_module_version_from_readme(modulename):
+    search_string = modulename + ":(.+)\n$"
+    with open("../../README.md", "r") as reader:
+        for line in reader:
+            result = re.search(search_string, line)
+            if result:
+                return result[1]
+
```

Comment:
I disagree. I think compatibility should only concern points of contact. We already test that prealignment and alignment of a certain version work together in the alignment module this would basically test the same thing. The output of alignment is the same for the same release and that is the only point of contact `qc` has so far.